### PR TITLE
Modified footer logos

### PR DESCRIPTION
**Pull request by AsavariA**
In accordance with issue #187 

and [issue #441](https://github.com/numpy/numpy.org/issues/441) of the [numpy.org](https://github.com/numpy/numpy.org) repository.

This PR solves this issue in this way - 

color - colorSecondary
color on hover - white(#fff)
size - 2.3em (which was previously 1.5em, anything greater than that seems too big and out of proportion)

image

### CODE REVIEW

**Review by jarrodmillman — APPROVED** [review]

**Comment by jarrodmillman** [comment]
Thanks! This LGTM. I would like to see if @stefanv has any suggestions before merging. So let's give this another day or so.

**Comment by jarrodmillman** [comment]
I think @InessaPawson @isabela-pf @MarsBarLee already approved this; but if at least one of you could review this, I would appreciate it.  Thanks!!

**Review by InessaPawson — APPROVED** [review]
LGTM. Thank you for working on this, @AsavariA!

**Comment by stefanv** [comment]
Thanks @AsavariA!

**Pull request merged**
